README.md

```diff
@@ -6,3 +6,18 @@
 CatColab (working name) is a collaborative environment for
 
 > formal, interoperable, conceptual modeling.
+
+To build locally, first, be sure you have wasm-pack installed. Clone the repository and run
```

Comment:
I'm hoping that a manual install of `wasm-pack` isn't necessary? It's in the dev dependencies for the frontend, which ought to suffice.

Reply:
Hm, yeah, I got it built first by installing it but let me try redoing this without it installed.

Reply:
Yeah, you're right.